tests/bootstrap.php

```diff
@@ -60,7 +60,7 @@ public function __construct() {
 		tests_add_filter( 'setup_theme', array( $this, 'install_wc' ) );
 
 		// Set up WC-Admin config.
-		tests_add_filter( 'wc_admin_get_feature_config', array( $this, 'add_development_features' ) );
+		tests_add_filter( 'woocommerce_admin_get_feature_config', array( $this, 'add_development_features' ) );
```

Comment:
Great catch, and interesting to see how this works in the test env ( pulling from the dev config that is ). Answered one of my questions from when I was working on this PR.